README.md

```diff
@@ -26,3 +26,11 @@ To push the latest versions to dockerhub just run:
 ```
 test.sh push
 ```
+
+To be able to test the OME-TIFF XML metadata, we need to install the packages that they use so we
+can run a Python test (directly diff-ing the files does not work because there is a uuid in the OMEXML).
+You may do this in a virtual environment or not.
+
+```
+pip install -r ./containers/ome-tiff-offsets/context/requirements.txt
+```
```

Comment:
I'd prefer not to go in this direction. I know that in vitessce-data Tos had the same problem last summer, and so we ran the output through sed first to normalize the part we couldn't control. Something like:
```
diff <(sed 's/re-for-something-that-varies/stable-string-to-match-fixture/' current-output) fixture
```